common/src/main/java/dev/lavalink/youtube/http/YoutubeOauth2Handler.java

```diff
@@ -213,35 +213,50 @@ public void refreshAccessToken(boolean force) {
                 return;
             }
 
-            // @formatter:off
-            String requestJson = JsonWriter.string()
+            try {
+                JsonObject json = createNewAccessToken(refreshToken);
+                updateTokens(json);
+                log.info("YouTube access token refreshed successfully");
+                log.debug("YouTube access token is {} and refresh token is {}. Access token expires in {} seconds.", accessToken, refreshToken, json.getLong("expires_in"));
+            } catch (Exception e) {
+                log.error("Failed to refresh access token", e);
+                throw e;
+            }
+        }
+    }
+
+
+    /**
+     * Executes the HTTP request to refresh the access token and returns the response.
+     * @param refreshToken The refresh token to be included in the request.
+     * @return The JSON response as a JsonObject.
+     */
+    public JsonObject createNewAccessToken(String refreshToken) {
+        String requestJson = JsonWriter.string()
                 .object()
-                    .value("client_id", CLIENT_ID)
```

Comment:
Please preserve the indents you have removed. That is why there is `@formatter:off` and `@formatter:on` lines as it better helps to visualize the object as it is being created.